### ISSUE TITLE
extended_properties storage seems to be valid up to SQL 2017 only

### ISSUE THREAD

**Issue by Guillaume-Fourrat** [MicrosoftDocs/sql-docs#2552]
extended_properties is used as storage of sensitivity metadata, as per current doc, up to SQL 2017.
SQL 2019 (as of CTP3) seems to use sys.sensitivity_classifications instead (not a mere wrapper since extended_properties is empty when sys.sensitivity_classifications is populated. Appears extended_properties exposes raw data from sysxprops when sensitivity_classifications uses sysobjvalues).

(I just had a look to closed entries and I believe this is the cause behind #2209)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 34863695-cd07-1260-4fea-66738c5fa80a
* Version Independent ID: 4c01015d-9c1d-db8e-84f5-379ffc5d5133
* Content: [SQL Data Discovery & Classification - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/security/sql-data-discovery-and-classification?view=sqlallproducts-allversions)
* Content Source: [docs/relational-databases/security/sql-data-discovery-and-classification.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/security/sql-data-discovery-and-classification.md)
* Service: **sql-database**
* GitHub Login: @barmichal
* Microsoft Alias: **mibar**

**Comment by AndreasWolter** [comment]
Yes, this is correct. The extended properties were merely used as a workaround for SQL 2017, because Data Classification was published as a client feature long after SQL 2017 shipped. This way it could still be integrated into the box product, whereas in Azure it was built-in from start

**Comment by VanMSFT** [comment]
Thanks @Guillaume-Fourrat and @AndreasWolter! I'll add a note for SQL Server 2017 for the existing example once the public PR merges.
